src/util/utiliof.c

```diff
@@ -2969,25 +2974,87 @@ iof * iof_filter_reader_replacement (iof *P, iof_handler handler, size_t statesi
   return F;
 }
 
+#ifdef _WIN32 /* --ak */
+#include <malloc.h>
+#include <sys/stat.h>
+#include <wchar.h>
+#include <windows.h>
+/*
+  Get wide string from multibyte string. (by T. Tanaka)
+*/
+static wchar_t *
+get_wstring_from_mbstring(int cp, const char *mbstr, wchar_t *wstr)
+{
+  int len;
 
+  len = MultiByteToWideChar(cp, 0, mbstr, -1, wstr, 0);
+  if (len==0) {
+    return NULL;
+  }
+  if (wstr==NULL) {
+    wstr = malloc(sizeof(wchar_t)*(len+1));
```

Comment:
IMO we should consequently use **util_malloc()**, as probably all code in pplib. afair there was two reasons to avoid direct calls to **malloc()**:
* easier replacement with some other base allocator (some day maybe)
* low-level check for NULL (and rather brutal solution..)

Reply:
I understand pplib defines util_malloc() at util/utilmem.c and is using util_malloc() instead of malloc().
I agree `util_malloc()` is better.